src/status_im/constants.cljs

```diff
@@ -415,14 +415,14 @@
 (def ^:const ens-action-type-set-pub-key 2)
 
 ;; wallet
-(def ^:const ethereum-chain-id 1)
-(def ^:const goerli-chain-id 5)
-(def ^:const arbitrum-chain-id 42161)
-(def ^:const arbitrum-testnet-chain-id 421613)
-(def ^:const optimism-chain-id 10)
-(def ^:const optimism-testnet-chain-id 420)
+(def ^:const ethereum-mainnet-chain-id 1)
+(def ^:const ethereum-goerli-chain-id 5)
 (def ^:const ethereum-sepolia-chain-id 11155111)
+(def ^:const arbitrum-mainnet-chain-id 42161)
+(def ^:const arbitrum-goerli-chain-id 421613)
 (def ^:const arbitrum-sepolia-chain-id 421614)
+(def ^:const optimism-mainnet-chain-id 10)
+(def ^:const optimism-goerli-chain-id 420)
 (def ^:const optimism-sepolia-chain-id 11155420)
```

Comment:
Aren't these changes already being done in another PR?

Reply:
Yes, This is much needed for this PR. I have included it. Not sure when the other PR will go through testing. 
We will fast-track this PR to get this merged.